### DISCUSSION TITLE
Is Ory Console open source?

### DISCUSSION THREAD

**Discussion by LasseRosenow** [ory/kratos#2850] [Q&A]
I see that in the repository is some talk about the Ory Console, but I can not find it anywhere, is it open source or is it for your cloud service only?

If it is not open source, are there any plans to open source it at some point in the future maybe?

**Comment by aeneasr** [comment]
Hi, the Ory Console is part of the commercial open source solution and not open source itself. Running on the Ory Network is cheaper than doing open source yourself (as it takes time and effort and lots of learning) so maybe give it a try! :)
